source/whats-new.txt

```diff
@@ -152,6 +152,9 @@ The 3.0 driver release includes the following new features:
   To learn more about using the aggregation pipeline with the {+driver-short+}, see
   :ref:`csharp-aggregation`.
 
+- Adds an API for bulk write operations. To learn more about bulk write operations, see
+  :ref:`csharp-bulk-write`.
```

Comment:
```suggestion
- Adds an API for bulk write operations at the client level. To learn more about bulk write operations, see
  :ref:`csharp-bulk-write`.
```

Reply:
Same question as above re client bulk writes

Reply:
There is a bulk write method at the collection level: https://mongodb.github.io/mongo-csharp-driver/3.0.0/api/MongoDB.Driver/MongoDB.Driver.IMongoCollection-1.BulkWrite.html

I think that v3 introduces "client bulk write", through a method at the client level: https://mongodb.github.io/mongo-csharp-driver/3.0.0/api/MongoDB.Driver/MongoDB.Driver.IMongoClient.BulkWrite.html

This enables you to include writes to multiple namespaces in a deployment, which is not possible in collection bulk write.

Reply:
Ah, I see--so "client" as opposed to "collection" rather than "server". good callout